### ISSUE TITLE
ability to filter games owned on steam

### ISSUE THREAD

**Issue by tombuben** [MestreLion/humblebundle#27]
I looked into it a bit and wrote some simple filtering using the python [steamapi](https://github.com/smiley/steamapi), but the naming convention for humble and steam are massively different.  
Steam only gives steamappID and human readable names, which are sometimes massively different from humble given names ("Trine 2: Complete Story" is just "Trine 2" on steam or "Q.U.B.E.: Director's Cut" is "Q.U.B.E: Director's Cut" on steam as examples).

Is there a way to get the steam appID from humble, considering humble checks if an user owns the game when redeeming the key?

**Comment by MestreLion** [comment]
I don't think this automatic matching is possible, as the JSON parsed from humble does not contain the  SteamID :( But, this could be manually entered at `gamedata.json`. Humble library is huge, but its bundles are not that many, so it is a viable community effort

**Comment by chilinux** [comment]
Trying to automate normalizing of strings between Humble Bundle and another app store is not really possible.  I tried to do something similar to match between android games from HB to add in metadata from the Google Play store entries for the same games.  I ended up having to do some of the normalizing manually by hand similar to the suggestion above to add it into the gamedata.json.

**Comment by basxto** [comment]
PcGamingWiki does have that data.
Maybe it would be best to start a database or list (in a git repo) for mapping games to different gaming platforms (humble, steam, gog, itch, …) and even tools (lutris, game backup monitor, …)
Data scrapped from pcgamingwiki could be a start.

The naming is partly a mess.
Humble: Shadow Warrior  and Shadow Warrior: Special Edition
Steam: Shadow Warrior, Shadow Warrior Classic (1997), Shadow Warrior (Classic) and Shadow Warrior Classic Redux
